src/Stoa.ts

```diff
@@ -1448,6 +1449,76 @@ class Stoa extends WebService {
             });
     }
 
+    /**
+     * GET /wallet/utxo/:address
+     *
+     * Called when a request is received through the `/utxo/:address` handler
+     *
```

Comment:
Parameters document is required.

Reply:
Other endpoints also have no criteria for that.